app/src/androidTest/java/com/android/unio/ui/event/EventCreationTest.kt

```diff
@@ -125,14 +122,3 @@ class EventCreationTest {
     assertDisplayComponentInScroll(composeTestRule.onNodeWithTag(EventCreationOverlayTestTags.SAVE))
   }
 }
-
-/**
- * This function is a copy of the function with the same name from EditAssociationTest.kt. It should
- * be extracted to a common file in a future PR.
- */
-private fun assertDisplayComponentInScroll(compose: SemanticsNodeInteraction) {
```

Comment:
This is not about your pull request, but we should remove all private `assertDisplayComponentInScroll` methods and use the one from the utils.

Reply:
You're right and this is done in my other PR connecting the event creation frontend to the backend